src/HallPIC.jl

```diff
@@ -455,4 +457,107 @@ function boltzmann_electric_field_and_potential!(E, phi, n_e, T_e, grid)
     return E, phi
 end
 
+#======================================================
+Reactions definitions
+======================================================#
+
+struct ReactingSpecies
```

Comment:
why not use the regular species struct above?

could then have a struct `coefficients::Vector{UInt8}` as a member of `Reaction`